explorer/interpreter/interpreter.cpp

```diff
@@ -576,8 +576,11 @@ auto Interpreter::StepLocation() -> ErrorOr<Success> {
 
 auto Interpreter::EvalRecursively(std::unique_ptr<Action> action)
     -> ErrorOr<Nonnull<const Value*>> {
+  auto action_kind = action->kind_string();
   if (trace_stream_->is_enabled()) {
-    *trace_stream_ << "--- recursive eval\n";
+    *trace_stream_ << "--- recursive eval for '" << action_kind << "'\n'''\n";
+    action->Print(trace_stream_->stream());
+    *trace_stream_ << "\n''' ---> \n";
```

Comment:
````suggestion
    *trace_stream_ << "--- recursive eval for '" << action_kind << "'\n```\n";
    action->Print(trace_stream_->stream());
    *trace_stream_ << "\n``` ---> \n";
````

May as well use markdown-like triple-quotes.